incentivizing_rewarding_participants.adoc

```diff
@@ -1,87 +1,157 @@
+Badge systems may therefore encourage contributors to act with greater self-interest—"to work towards the badge without focusing on the well-being of the community."
+Snell recommends clearly documenting badge requirements and submitting new commnity badges to a mandatory review (or "incubation") period before implementing them.
+
+=== Community-branded items
+
+You might reward contributors by gifting them various items that signify their affiliation with the community.
+For example, you might consider giving shirts, hats, stickers, or keychains featuring the project's logo to participants in order to enhance their sense of connection with the project.
+If your community features a leaderboard-style reward model (see previous section), it might tie receipt of a particular branded item to some achievement on that board ("receive a community-branded t-shirt when you've collected 100 participation points").
+This reward model is especially useful when you want to connect with contributors who derive a strong sense of identity from participating in the project and associating with its community.
+In this way, receiving items that allow them to publicly express their affiliation with an important peer group—or signal their status _within_ that group—might motivate them to contribute to a project.
+
+This reward model often requires substantial resources, however. The work of packing and shipping branded items to members of a global community can be time-consuming.
+Moreover, production and distribution of branded materials can involve significant monetary expenses.
+
+=== Community spotlights and awards
+
+Community spotlights and awards help you showcase the people who make your community great.
+Some open source communities publish occasional blog posts or run video series to spotlight contributors who have made significant impacts on the project.
+For instance, they might feature a monthly "Meet the Contributors" or "Community Spotlight" series that highlights key contributors, allowing them to tell the community more about themselves.
+Other communities might develop an annual award series, recognizing a "Contributor of the Year" or "Most Valuable Maintainer."
+The project might then feature award winners at events or in featured interviews posted online.
+
+Because this reward model involves opportunities for gaining visibility, it's useful when you know participants are motivated by a desire to enhance their personal reputations and grow their professional networks.
+The model is also particularly useful for attracting and retaining diverse contributors.
+Communities spotlighting contributors from under-represented groups provide a way for new, potential contributors from those groups to "see themselves" in their projects and feel more confident joinging them.
```

Comment:
The phrase "in their projects' is a bit confusing, as the subject are people not yet in the project looking to see themselves there. I think the solution is to make it "the project"? Or I'm missing the subject:object connection properly.

typofix:  s/joinging/joining/

Reply:
Great catch. It was an errant word. Fixed!